src/adminjs-options.interface.ts

```diff
@@ -415,6 +421,25 @@ export type PageHandler = (
   context: PageContext,
 ) => Promise<any>
 
+/**
+ * Bundle options
+ *
+ * @alias BundlerOptions
+ * @memberof AdminBroOptions
```

Comment:
Could you change that to `AdminJSOptions`? And update the example below as well.